src/tests/codecs/test_EtcPakCodec.py

```diff
@@ -57,12 +59,23 @@ def test_decompress_DXT5_succeeds():
             assert expected.getpixel(coords) == actual.getpixel(coords) # type: ignore[reportUnknownMemberType]
 
 
+@pytest.mark.skipif(os.name == "nt", reason="EtcPak ETC1 decompress segfaults on windows")
 @pytest.mark.codecs
 @pytest.mark.codecs_ETC1
-def test_decompress_ETC1_succeeds():
+def test_decompress_ETC1_succeeds_onLinux():
     with smileyRoundtripImage(CompressionFormat.ETC1) as expected:
         compressed = SMILEY_COMPRESSED_RAW[CompressionFormat.ETC1]
         actual = CODEC.decompress(compressed, CompressionFormat.ETC1, expected.width, expected.height, None) \
             .convert(expected.mode)
         for coords in zip(range(expected.width), range(expected.height)):
             assert expected.getpixel(coords) == actual.getpixel(coords) # type: ignore[reportUnknownMemberType]
```

Comment:
we swap the bytes at decompression. this is not accounted for in this test. 
I assume the pixel order in the compressed is wrong since the swapping to BGR is not actually mandated by the ETC standart. It seems to be just an implementation flavor of etcpak. I think this fails since AEI_Editor also doesn't swap the color channels which suggests that .aei actually stores ETC1 in RGB instead of BGR. But i am not entirely sure.
```suggestion
            pixel = actual.getpixel(coords)
            assert expected.getpixel(coords) == (pixel[2],pixel[1],pixel[0]) # type: ignore[reportUnknownMemberType]
```